pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -115,8 +115,12 @@ func (r *REST) Update(ctx kapi.Context, tagName string, objInfo rest.UpdatedObje
 			return nil, false, kapierrors.NewBadRequest("namespace is required on ImageStreamTags")
 		}
 		imageStream = &api.ImageStream{
-			ObjectMeta: kapi.ObjectMeta{Namespace: namespace, Name: name},
+			ObjectMeta: kapi.ObjectMeta{
+				Namespace: namespace,
+				Name:      name,
+			},
 		}
+		kapi.FillObjectMetaSystemFields(ctx, &imageStream.ObjectMeta)
```

Comment:
This is being done to fake the new `UpdatedObject` call that @liggitt added?  There's got to be a better way to handle create on update calls.